<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCad_Livro</class>
 <widget class="QDialog" name="frmCad_Livro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Livro</string>
  </property>
  <widget class="QPushButton" name="concluirPushButton">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>231</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Concluir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alterarPushButton">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>231</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>90</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="isbnLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ISBN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="isbnLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>10</y>
     <width>190</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="tituloLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Titulo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tituloLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>90</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="anoLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ano</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="anoSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>70</y>
     <width>270</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="autorLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Autor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="autorComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>126</y>
     <width>270</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="editoraLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Editora</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="editoraComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="incluirAutorPushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>93</y>
     <width>102</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Incluir Autor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="incluirEditoraPushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Incluir Editora</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>182</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exemplares</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="exemplaresSpinBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
